src/input/keyboard.c

```diff
@@ -407,7 +409,18 @@ handle_compositor_keybindings(struct keyboard *keyboard,
 		keyinfo.is_modifier);
 
 	if (event->state == WL_KEYBOARD_KEY_STATE_RELEASED) {
-		return handle_key_release(server, event->keycode);
+		if (cur_keybind && cur_keybind->mod_only) {
+			if (seat->active_client_while_inhibited
+					|| seat->server->session_lock) {
+				cur_keybind = NULL;
```

Comment:
We store key as bound (https://github.com/labwc/labwc/blob/1cd9940307c4b54503cefcd643e92314ade88c45/src/input/keyboard.c#L454) so need to call `key_state_bound_key_remove()` here.

Reply:
Done